specification/ml/_types/TrainedModel.ts

```diff
@@ -76,28 +79,38 @@ export class TrainedModelDeploymentStats {
    */
   nodes: TrainedModelDeploymentNodesStats[]
   /** The number of allocations requested. */
-  number_of_allocations: integer
+  number_of_allocations?: integer
+
+  peak_throughput_per_minute: long
+
+  priority: TrainingPriority
   /** The number of inference requests that can be queued before new requests are rejected. */
-  queue_capacity: integer
+  queue_capacity?: integer
   /**
    * The sum of `rejected_execution_count` for all nodes in the deployment.
    * Individual nodes reject an inference request if the inference queue is full.
    * The queue size is controlled by the `queue_capacity` setting in the start
    * trained model deployment API.
    */
-  rejected_execution_count: integer
+  rejected_execution_count?: integer
   /** The reason for the current deployment state. Usually only populated when
    * the model is not deployed to a node.
    */
-  reason: string
+  reason?: string
   /** The epoch timestamp when the deployment started. */
   start_time: EpochTime<UnitMillis>
   /** The overall state of the deployment. */
   state: DeploymentAssignmentState
```

Comment:
Per https://github.com/elastic/elasticsearch/blob/045f6a31f994f51d87a217be60251e060132c8a1/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/assignment/AssignmentStats.java#L598-L600:

```suggestion
  state?: DeploymentAssignmentState
```